src/mits_s/config.849

```diff
@@ -17,7 +17,7 @@
 
 .if z %opsys-"IT		;if under ITS
 	.macro $insrt fn1,fn2
-		.insrt MC:MITS.S;fn1 fn2
+		.insrt MITS.S;fn1 fn2
```

Comment:
Veeery picky comment, but: do the version bump as part of this commit?

Reply:
Thanks, fixed.